setup.py

```diff
@@ -28,8 +28,8 @@
         "requests",
         "sentry-sdk[celery,sqlalchemy]",
         "sqlalchemy>=1.4",
-        "starlette",
-        "typing-extensions",
+        "starlette==0.17.1",
+        "typing-extensions==4.0.1",
```

Comment:
Can we add a comment as to why these need to be pinned?

Reply:
Everything is pinned, these were just missed last time.  Comment is here:

https://github.com/microbiomedata/nmdc-server/blob/0c0c8bb42df29ffad19deafa1a2dc19ca5890259/setup.py#L9-L10

Reply:
They don't.  Unpinned in https://github.com/microbiomedata/nmdc-server/pull/582/commits/7da8c6b403ca7a3eda2a885d6ab52188a17002ec